src/main/java/org/gridsuite/modification/modifications/LccCreation.java

```diff
@@ -0,0 +1,212 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.modification.modifications;
+
+import com.powsybl.commons.report.ReportNode;
+import com.powsybl.commons.report.TypedValue;
+import com.powsybl.iidm.network.*;
+import org.gridsuite.modification.NetworkModificationException;
+import org.gridsuite.modification.dto.LccConverterStationCreationInfos;
+import org.gridsuite.modification.dto.LccCreationInfos;
+import org.gridsuite.modification.utils.ModificationUtils;
+import org.gridsuite.modification.utils.PropertiesUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.gridsuite.modification.NetworkModificationException.Type.CREATE_LCC_ERROR;
+import static org.gridsuite.modification.NetworkModificationException.Type.HVDC_LINE_ALREADY_EXISTS;
+import static org.gridsuite.modification.utils.ModificationUtils.createInjectionInNodeBreaker;
+import static org.gridsuite.modification.utils.ModificationUtils.reportInjectionCreationConnectivity;
+
+/**
+ * @author Rehili Ghazwa <ghazwa.rehili at rte-france.com>
+ */
+
+public class LccCreation extends AbstractModification {
```

Comment:
I think it should be hvdc and shared with vsc

Reply:
yes